notfound/extension.py

```diff
@@ -49,11 +49,13 @@ def pathto(otheruri, resource=False, baseuri=None):
         if not resource:
             otheruri = app.builder.get_target_uri(otheruri)
 
-        if baseuri is None:
+        if baseuri is None and not app.config.notfound_gh_pages:
             baseuri = '/{language}/{version}/'.format(
                 language=app.config.notfound_default_language,
                 version=app.config.notfound_default_version,
             )
+        elif baseuri is None:
```

Comment:
nit: I'd rewrite this if like

```
if baseuri is none:
  if gh_pages:
    bsaeuri = /
  else:
    baseuri = /language/version
```

I think it's easier to read.

Reply:
Resolved in ef237e9